com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -385,6 +385,7 @@ public IReadOnlyList<ulong> ConnectedClientsIds
         public string ConnectedHostname { get; private set; }
 
         internal INetworkMetrics NetworkMetrics { get; private set; }
+        private int m_LastMetricsTickId;
```

Comment:
Is there a default initialization or is that a risk to have garbage value?

Reply:
Good point, I think I need to reset this on server init in case a network manager is reused.

Reply:
You were right. There was a very specific edge case where, if a network manager had executed exactly 1 tick, dispatched its metrics, and then was shutdown, it would not dispatch its first frame of metrics after the next startup. Wrote a test to cover this case.